### PR TITLE
fix: move from redis db to mongodb

### PR DESCRIPTION
**Pull request by Azat2202**
Features: 
- Use MongoDb instead of Redis
- MongoDb is implemented in async way
- Enabled logging

Fixes:
- Answers containing "like" emoji like "ШУЛЬГА👍👍👍👍" are striped

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ services:
   main-app:
```
Comment (Arlet2) [inline compose.yaml:3]: add restart: always

File: compose.yaml [hunk 2]
```diff
@@ -3,6 +3,8 @@ services:
...
     image: arlet2/vt_polls_bot
     env_file: .env
+    depends_on:
+      - mongo
 
   watchtower:
```
Comment (Arlet2) [inline compose.yaml:9]: add restart: always

File: compose.yaml [hunk 3]
```diff
@@ -11,14 +13,14 @@ services:
...
     command: --interval 30
 
-  redis:
-    image: redis:7.2.4-alpine
-    restart: unless-stopped
-    volumes:
-      - redis:/data
-    command: redis-server --save 60 1 --loglevel warning
+  mongo:
+    image: mongo
```
Comment (Arlet2) [inline compose.yaml:17]: add version (7.0.14 in example)

File: handlers/quiz.py [hunk 4]
```diff
@@ -12,23 +12,12 @@
...
-
-
-def create_poll_message(quiz_name: str, quiz_answers: dict[str, int]) -> str:
-    return f"quiz: {quiz_name}\n" + "\n".join([
-        f"- {answer} " + "👍" * (int(count) - 1)
-        for answer, count
-        in sorted(quiz_answers.items(), key=lambda x: x[1], reverse=True)
+def create_poll_message(quiz: dict) -> str:
+    return f"quiz: {quiz['name']}\n" + "\n".join([
+        f"- {option['name']} " + "👍" * int(option['value'])
```
Comment (Arlet2) [inline handlers/quiz.py:18]: Glory to the UTF-8

File: compose.yaml (hunk 1 above)
Comment (Azat2202) [reply compose.yaml:3]: у тебя у самого не было)

**Pull request merged**